### DISCUSSION TITLE
Getting back the Google Authenticator after reseting Iphone

### DISCUSSION THREAD

**Discussion by Hoang1135** [Authenticator-Extension/Authenticator#844] [Q&A]
I have just reseting my Iphone for some problems. And I erased all the applications of my phone including Google Authenticator. 
All of my 2 verification for Google and Social Media were in previous Application Google Authenticator. After losing all the 2 verification in Google Authenticator, i cannot sign in into any account in my phone without Authenticator. 

So may I ask how can i get back or restore the Authenticator Number of all my Account?

**Comment by Sneezry** [comment]
Hi @Hoang1135 , this project is for Authenticator browser extension, it has no relationship with Google's mobile App.
